joomla/joomla-cms
# [6.0] Added possibility to batch remove a tag

Pull Request for Feature request #23185 .

### Summary of Changes
In the articles view there is a batch action which can add/assign one tag from the tag list to the selected articles. This PR introduces the possibility to remove/unassign one tag from the selected articles. It also extends batch processing for categories, contacts and news feeds.

How it looks like:

![articles-batch-tag-removing2](https://github.com/joomla/joomla-cms/assets/8353850/d637b2e6-96a9-4200-96f4-592299f2854a)


### Testing Instructions
If you use patch tester, after applying the patch, call `npm install` from commandline to rebuild the .js files in folder media.

1. Make sure your Joomla tag system is active and the plugin "Behaviour - Taggable" is enabled.
2. Create a new article and in the detail view add/assign two tags, lets say "Tag1" and "Tag2".
3. Save and create a second article, add/assign it to tag "Tag2" and "Tag3".
4. Save and close.
5. In the articles list select both articles, click batch action and remove "Tag2". Apply batch processing.
6. Open the first article details and check whether the "Tag2" is removed. The tag "Tag1" must be still assigned.
7. Open the second article details and check whether the "Tag2" is removed. The tag "Tag3" must be still assigned.
8. A similar test should be done for categories, contacts and news feeds.


### Actual result BEFORE applying this Pull Request
It is not possible to remove/unassign a tag from an article via batch processing.


### Expected result AFTER applying this Pull Request
It is possible to remove/unassign a tag from an article via batch processing.


### Link to documentations
Please select:


There are new text keys that need to be translated.
